test/integration/__tests__/live-api.js

```diff
@@ -293,9 +294,13 @@ describe('API integration', function () {
       const client = createClient()
 
       let progressCalled = false
-      function onUploadProgress ({ uploadedBytes }) {
+      function onUploadProgress ({ uploadedBytes, totalBytes }) {
         // console.log(uploadedBytes)
         expect(uploadedBytes).toBeDefined()
+        if (isResumable) {
```

Comment:
these seem unrelated :thinking:

Reply:
Yes. I discovered some issues/improvements while converting to the new eslint config. I can pull those commits out to another branch

Reply:
No need as long as you are aware :ok_hand: